cookbooks/ruby/recipes/default.rb

```diff
@@ -50,7 +50,7 @@
     else
       echo "Installing Ruby #{ruby_version}"
       mkdir -p /opt/rubies
-      chown #{node['owner_name']}:#{node['owner_name']} /opt/rubies
+      chown #{node['owner_name']}:#{node['owner_name']} /opt/rubies -R
```

Comment:
According to https://linux.die.net/man/1/chown, the flag should immediately follow the `chown` command.

Reply:
Fixed.